03-specifying-sample-designs.Rmd

```diff
@@ -635,8 +635,8 @@ library(tidycensus)
 
 pums_in <- get_pums(
   variables = c("NP", "BDSP", "HINCP"),
-  state = "37",
-  puma = c("01301", "01302"),
+  state = "37", 
```

Comment:
Do we want to reference these in the paragraph above when we talk about them.  People may not know what these are.